content/es/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Recolección de Basura
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+El papel del recolector de basura de Kubernetes es el de eliminar determinados objetos
+que en algún momento tuvieron un propietario, pero que ahora ya no.
+
+{{% /capture %}}
```

Comment:
This doesn't look right - that shortcode won't render on the _master_ branch because PR https://github.com/kubernetes/website/pull/21359 removed `capture` shortcodes.

Reply:
That probably was the issue, as you already saw, I created https://github.com/kubernetes/website/issues/22168 to review as this affects to several pages. Thanks for taking your time to spot this!